public_dropin_environments/python3_sklearn_fedramp_compliant/env_info.json

```
@@ -0,0 +1,15 @@
+{
+  "id": "679b441e9c77601e2fadd45e",
+  "name": "[DataRobot] Python 3.9 Scikit-Learn Drop-In [FedRAMP-compliant]",
```

```suggestion
  "name": "[DataRobot][FedRAMP-compliant] Python 3.9 Scikit-Learn Drop-In",
```

I feel like federal customers will want to only use compliant images so better to have that information first?